parlai/crowdsourcing/utils/tests.py

```diff
@@ -186,6 +213,25 @@ def _register_mock_agents(
 
         return agent_ids
 
+    def await_channel_requests(self, timeout=2) -> None:
+        time.sleep(0.1)
+        tracked_run = self._get_live_run()
+        self.assertTrue(  # type: ignore
+            self.operator._run_loop_until(
+                lambda: len(tracked_run.client_io.request_id_to_channel_id) == 0,
+                timeout,
+            ),
+            f"Channeled requests not processed in time!",
+        )
+
+    def assert_sandbox_worker_created(self, worker_name, timeout=2) -> None:
+        self.assertTrue(  # type: ignore
```

Comment:
Note, this is failing on `in_flight_qa` tests because it's not extending a test case presumably? You may want to rewrite these two helpers to just throw an assertion error instead.

Reply:
Interesting, yeah perhaps

Reply:
These checks seem to be passing now